docs/connected_orgs.rst

```diff
@@ -1,14 +1,76 @@
-Connect persistent orgs
+Connect Persistent Orgs
 =======================
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you first need to have a Connected App configured in a persistent Salesforce org.
 
-Connect a production org
-------------------------
 
-Connect a sandbox org
----------------------
 
-Global orgs
------------
+The ``org connect`` Command
+---------------------------
+To connect to a persistent org use::
 
-Use a custom connected app
+    $ cci org connect <org_name>
+
+This will automatically open a browser window pointed to a Salesforce login page.
+The provided ``<org_name>`` will be the name that CumulusCI associates with org you log into.
+
+If you're connecting to a sandbox use the ``--sandbox`` flag.
```

Comment:
I want to call this out in its own section (hence the separate "Connect a sandbox org" header removed in the diff here) to reduce the chance that it gets overlooked.

We should also spell out the different types of orgs for each way (i.e. without the sandbox flag is for production orgs and Developer Edition orgs. With the sandbox flag is for sandboxes and scratch orgs if you want to connect a scratch org that someone else created)

We should also mention the --login-url option.

Reply:
@davidmreed - You had mentioned collapsing these sections previously, do you have issues with going back to a separate subheaders for "Sandboxes"? 

I think that the subheaders also serve a second function as the new docs theme were using uses the section headers in a "contents" section on the right hand side bar. This means that if I'm a user looking to connect a persistent org, I can immediately click on the header that pertains to the type of org I'm looking for.

Reply:
I think it's fine. I would suggest that we mention that `--sandbox` would also be used for a scratch org you didn't create.